statsmodels/base/model.py

```diff
@@ -931,6 +931,14 @@ def initialize(self, model, params, **kwd):
         if hasattr(model, 'k_constant'):
             self.k_constant = model.k_constant
 
+    @cache_readonly
+    def fittedvalues(self):
+        return self.predict()
```

Comment:
No, from inside the results class `self` is without wrapping, so it's always numpy ndarray
I made a comment about this not inline.

Reply:
I'll look again, but as I recall the `Results.predict` method doesn't conform to the usual patterns.

Reply:
After looking again, I'm still under the same impression I was before.  Am I more confused than usual?

Reply:
I think we are talking about different things
`return self.predict()` is internal and returns numpy
`fittedvalues` itself is wrapped (AFAIR), so it will be pandas from the user side.
`self` in the model is without wrapper, but the user gets the wrapped instance.

Reply:
> I think we are talking about different things

This seems entirely plausible; I appreciate you taking the time to help sort through my confusion.

My claim is that the _unwrapped_ `Results.predict` method returns a pandas object (assuming `PandasData`), as a result of which the unwrapped `Results.fittedvalues` will be a pandas object.  Not that this is a problem per se, just not readers/contributors would usually expect.